src/main/java/be/syntra/devshop/DevshopBack/services/ProductService.java

```diff
@@ -0,0 +1,9 @@
+package be.syntra.devshop.DevshopBack.services;
+
+import be.syntra.devshop.DevshopBack.models.ProductDTO;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
the annotation should only be on implementation(s) of the interface not in the interface itself.